test/ring/middleware/test/refresh.clj

```diff
@@ -1,6 +1,88 @@
 (ns ring.middleware.test.refresh
-  (:use clojure.test
-        ring.middleware.refresh))
+  (:require
+   [clojure.java.io :as io]
+   [clojure.test :refer [are deftest testing]]
+   [ring.middleware.refresh :as sut]))
 
-(deftest test-wrap-refresh
-  (is (true? false)))
\ No newline at end of file
+(defn create-success-html-handler [body]
+  (constantly {:body body, :headers {"Content-Type" "text/html"}, :status 200}))
+
+(defn invoke-with-get-request [handler]
```

Comment:
Maybe:

```clojure
(defn- get-test-response-body [handler]
  (:body (handler {:request-method :get, :uri "/"})))
```

Again, the reasoning is that we don't need anything except the body.

Reply:
Done.